planning/planning_test_utils/include/planning_test_utils/planning_test_utils.hpp

```diff
@@ -359,32 +440,32 @@ void setSubscriber(
     test_node, topic_name, [&count](const typename T::ConstSharedPtr) { count++; }, subscriber);
 }
 
-void spinSomeNodes(
-  rclcpp::Node::SharedPtr test_node, rclcpp::Node::SharedPtr target_node,
-  const int repeat_count = 1)
-{
-  rclcpp::executors::SingleThreadedExecutor executor;
-  executor.add_node(test_node);
-  executor.add_node(target_node);
-  for (int i = 0; i < repeat_count; i++) {
-    executor.spin_some(std::chrono::milliseconds(100));
-    rclcpp::sleep_for(std::chrono::milliseconds(100));
-  }
-}
-
+/**
+ * @brief Publishes data to a target node.
+ *
+ * This function publishes data to a target node on a given topic, ensuring that the topic has
+ * subscribers and retrying a specified number of times.
+ *
+ * @tparam T The type of the message to publish.
+ * @param test_node The node to create the publisher on.
+ * @param target_node The target node to publish the message to.
+ * @param topic_name The name of the topic to publish to.
+ * @param publisher A shared pointer to the publisher.
+ * @param data The data to publish.
+ * @param repeat_count The number of times to retry publishing.
+ */
 template <typename T>
 void publishToTargetNode(
   rclcpp::Node::SharedPtr test_node, rclcpp::Node::SharedPtr target_node, std::string topic_name,
   typename rclcpp::Publisher<T>::SharedPtr publisher, T data, const int repeat_count = 3)
 {
   if (topic_name.empty()) {
-    int status;
+    int status = 1;
```

Comment:
very small question for my future code,
What's the motivation for this?

Reply:
It's always a good practice to initialize value.